expression/builtin_encryption.go

```diff
@@ -580,9 +583,58 @@ type builtinSHA2Sig struct {
 	baseBuiltinFunc
 }
 
+// Supported hash length as the second argument
+const (
+	SHA0   int = 0
+	SHA224 int = 224
+	SHA256 int = 256
+	SHA384 int = 384
+	SHA512 int = 512
+)
+
 // See https://dev.mysql.com/doc/refman/5.7/en/encryption-functions.html#function_sha2
 func (b *builtinSHA2Sig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("SHA2")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	// SHA2 function accepts 2 parameters
```

Comment:
Length of arguments would be checked in `verifyArgs` before,
so we don't need to check it here.